cmd/createtree/main.go

```diff
@@ -151,24 +152,30 @@ func newPK(opts *createOpts) (*any.Any, error) {
 			Password: opts.pemKeyPass,
 		}
 		return ptypes.MarshalAny(pemKey)
+	case "PKCS11ConfigFile":
+		if opts.pkcs11ConfigPath == "" {
+			return nil, errors.New("empty PKCS11 config file path")
+		}
+		return ptypes.MarshalAny(&keyspb.PKCS11ConfigFile{opts.pkcs11ConfigPath})
```

Comment:
I originally started with the contents of the PKCS#11 config in the proto instead of the path but decided not to as it would still end up containing a path that would need to exist on all systems (the path to the [PKCS#11 module](https://godoc.org/github.com/letsencrypt/pkcs11key#Config) to load). My thought is that in that case it makes sense to store the highest level path (to the config) so that machines with the modules in different locations could just modifying the local config in order to point to the right place.

In reality though it is likely that the module will exist in the same place on all systems (just as the config would) so either way it won't _really_ make that much of a difference. If you have strong opinions on the matter I'm happy to go either way ¯\_(ツ)_/¯.